Sources/Lux/Extensions/NSRegularExpression+Extensions.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+extension NSRegularExpression {
+
+    /// Returns the range of the first string inside double quotes in the given string
+    static func rangeOfQuotedString(in string: String) -> NSRange? {
+        let regex = try! NSRegularExpression(pattern: #"(?<=")[^"]+(?=")"#, options: [])
```

Comment:
I'm glad I haven't seen that... oh wait...

Reply:
I know you don't like the ` try!` but when it comes to `NSRegularExpression` initialisation, I am a bit tired to conditionally make one when I tested it before. In this case, we return a conditional result, but in other cases the regexes are static var and it would be painful to conditionally make them although we know it cannot fail.